test/output/only-nested/expected.js

```diff
@@ -0,0 +1,3 @@
+"use strict";function f() {var a,c;module.importSync("./module",{"a":function(v){a=v},"b":function(v){c=v}},0);
+
+}
```

Comment:
Ahhh so if we later toggle nested imports then this can be skipped when toggled off?